pkg/images/images_test.go

```diff
@@ -0,0 +1,337 @@
+package images
+
+import (
+	"testing"
+
+	"github.com/rancher/rke/types"
+)
+
+func Test_toKeep(t *testing.T) {
```

Comment:
This test case is good enough for now but hard to cherry-pick to the future release like `dev-v2.8`. It would be nicer if it is portable.

Reply:
good call. I think it is acceptable because it will be a one-time effect and there will be some other variables, like the min and max Rancher versions, that need to be changed for `dev-v2.8` anyway. 

I have not found an easy way to make them portable so far. But I am open to discussions if anyone has any ideas.